test/e2e/apps/job.go

```diff
@@ -361,6 +367,52 @@ var _ = SIGDescribe("Job", func() {
 			framework.ExpectEqual(pod.Status.Phase, v1.PodFailed)
 		}
 	})
+
+	ginkgo.It("should run a job to completion with cpu requests", func() {
```

Comment:
Perhaps we need to add a [Serial] here to address https://github.com/kubernetes/kubernetes/pull/108366#issuecomment-1064547192 so this test don't run in parallel

Reply:
Good point